airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/constants/GlobalDataSizeConstants.java

```diff
@@ -9,12 +9,12 @@
 
 public interface GlobalDataSizeConstants {
 
-  /** 256 MB to BYTES as comparison will be done in BYTES */
-  int DEFAULT_MAX_BATCH_SIZE_BYTES = DataSize.of(256L, IEC.MEBIBYTE).toUnit(IEC.BYTE).getValue().intValue();
+  /** 25 MB to BYTES as comparison will be done in BYTES */
+  int DEFAULT_MAX_BATCH_SIZE_BYTES = DataSize.of(25L, IEC.MEBIBYTE).toUnit(IEC.BYTE).getValue().intValue();
```

Comment:
Yes, we can control the file size with these constants. In our case, this is DEFAULT_MAX_BATCH_SIZE_BYTES (25 MB) * number of chunks (4) + buffer size - the size of one record. On my test data, this is 120.4 MB. In all other cases, it will not exceed 125 MB. Yes, you are right, it will affect the S3 staging files for redshift. But, to be honest, I don't see a problem with that. I tested redshift too, on a 13GB database and it worked well.

Reply:
If this does not suit us, we can change the MAX_PARTS_PER_FILE for redshift to any other number. For example, if we want a staging file size of 250MB, we can change it from 4 to 10. If we want 1GB, we change it to 40.